test/test_css_parser_loading.rb

```diff
@@ -197,10 +197,13 @@ def test_suppressing_circular_reference_exceptions
   def test_toggling_not_found_exceptions
     cp_with_exceptions = Parser.new(:io_exceptions => true)
 
-    assert_raises RemoteFileError do
+    err = assert_raises RemoteFileError do
       cp_with_exceptions.load_uri!("#{@uri_base}/no-exist.xyz")
     end
 
+    uri_regex = Regexp.new(Regexp.escape("#{@uri_base}/no-exist.xyz"))
+    assert_match uri_regex, err.message
```

Comment:
This came from [ed148aa](https://github.com/premailer/css_parser/commit/ed148aab13c142bd3daf8160e82d6644ce29478e#diff-ef9685ca9e71df32025876c2e7ee6535R204)